api/db/changelog/db.changelog-166-add-access-module-tables.xml

```diff
@@ -0,0 +1,55 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<databaseChangeLog
+  xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+  xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+  xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog
+                      http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.1.xsd">
+  <changeSet author="yonghao" id="changelog-166-add-access-module-tables">
+    <createTable tableName="access_module">
+      <column name="access_module_id" type="bigint" autoIncrement="true">
+        <constraints primaryKey="true" nullable="false"/>
+      </column>
+      <column name="access_module_enum" type="VARCHAR(80)">
```

Comment:
I thought what you were proposing was using the `enum` type in MySQL. I probably prefer integer over string type. That said, I'm not sure these are really enums... they seem more like constant IDs.

Reply:
You mean having constant IDs(which can just be the PK - access_module_id) and put the constant id -> human readable module name in code? Then that comes to the same concern: Have to lookup enum code to understand what the integer means. I'd like to see them being present in DB. 
enum seems to be a better choice than String. (One reason comes from here https://github.com/DataBiosphere/terra-resource-janitor/pull/6#discussion_r447249579 with `remarks`, and [tests](https://github.com/DataBiosphere/terra-resource-buffer/blob/2945375913472a6524b5f5453cb33ce60392dff4/src/test/java/bio/terra/buffer/common/BackwardsCompatibilityTest.java#L67L73) ). 

What would you prefer: 
1. integer? If so, I can just drop this and use access_module_id? 
2. enum?: Is that natively supported in liquibase
3. String?: It might be easier for my examoke above because it uses JDBC, and we wrote our own RowMapper. I don't know how to plug string-enum convert magic into JPA magic world.

-

Reply:
My preference is:

1. enum (IF the Hibernate and Liquibase integration look OK)
2. integer (status quo)
3. string

Reply:
What about "access_module_name" for this field, or simply "name" ?

Reply:
@jmthibault79 just to confirm if I understand this correctly. Are you also suggesting to make this regular string (name seems will be a regular string like user name, like institution's `displayName`)? Or this still should be an enum?

Reply:
summary: Now I use `enum` type, column name is `name`, and it is `PK` of the table

Reply:
keep it an enum